ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml

```diff
@@ -1,4 +1,76 @@
 periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 8 */2 * *
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: redhat-chaos
+    repo: krkn-hub
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: ARM
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-chaos-krkn-hub-main-ARM-krkn-hub-tests
+  spec:
```

Comment:
lets be sure to try to add the slack integration here

Reply:
Done! Thanks!

Reply:
Missed it, should be in now! Thanks.